### DISCUSSION TITLE
Is it possible to install openCravat to different place other than C:/ in windows

### DISCUSSION THREAD

**Discussion by WeiMeihan** [KarchinLab/open-cravat#253] [Usability & Feature Requests]
I have no memory in my computer hard-drive C:/ but the openCravat is automatically installed in C:/. It does not give me an option to change the installing directory.

I also tried to move whole "opencravat" folder to extra hard drive. It seems the pathways mix up and does not work.

Could you please help me figure out how to install in different place in windows?

Thank you in advance!

**Comment by rkimoakbioinformatics** [comment]
Hi, after installation, click the OpenCRAVAT icon to open it, and then, on the browser tab that opens, click the settings icon at the top right, and enter the new path for OpenCRAVAT modules at "Modules dir" and click "Save". Then, exit the OpenCRAVAT app and open it again. Then, install the system modules if needed or if you already installed the system modules, move the system modules file to the new path. Let me know if this works for you.
